### DISCUSSION TITLE
Application's indicator (tray icon) not appeared on Xfce4

### DISCUSSION THREAD

**Discussion by directentis1** [xushengfeng/eSearch#235] [Q&A] [answered]
As the question, I've already installed these packages on my Ubuntu 22.04, but still can't see eSearch trayicon.

![image](https://github.com/xushengfeng/eSearch/assets/128832114/2695fd9a-fa99-4ec1-9002-87cec15b8500)

Any suggestion for me? 😄

**Comment by xushengfeng** [comment]
Can the tray icons of other app display properly?

**Reply by directentis1** [reply]
Yes they show up normally.

 Just a bit of trouble recently where I had to reset the panel (xfce4-panel -r) to see them.

**Reply by directentis1** [reply, marked as answer]
Hey, I got solved.
Some bugs related of ayatana-indicator-application package on Xfce4.
When I removed that and kill it, my taskbar works normally.

[\[SOLVED\] xfce4-panel Requires Refresh for All Icons to Appear / Desktop / Xfce Forums](https://forum.xfce.org/viewtopic.php?id=16436)
